ateam_bringup/launch/bringup_simulation.launch.py

```diff
@@ -59,7 +59,7 @@ def generate_launch_description():
 
     ui_path = os.path.join(get_package_share_directory('ateam_ui'), 'launch')
     ui_launch = launch.actions.IncludeLaunchDescription(
-        PythonLaunchDescriptionSource([ui_path, '/ateam_ui_launch.py']))
+        PythonLaunchDescriptionSource([ui_path, '/ateam_ui_debug_launch.py']))
```

Comment:
This is an accidental change, but everyone in the hotel room seemed to like it, so I'll keep it in this PR.